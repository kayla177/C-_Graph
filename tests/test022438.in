ENTITY JQ753OIK name_JQ75 paper
ENTITY 4QREPUTO name_4QRE author
ENTITY ZHZYL5A9 name_ZHZY institution
ENTITY Y0AWLUUN name_Y0AW paper
ENTITY J45F3NWR name_J45F institution
ENTITY 2FU3NZR8 name_2FU3 journal
ENTITY 3BURPWXO name_3BUR author
ENTITY UERCIK82 name_UERC institution
ENTITY SXA506HI name_SXA5 conference
ENTITY UOGODPMX name_UOGO journal
RELATIONSHIP ZHZYL5A9 cited_by 3BURPWXO 2.0
RELATIONSHIP J45F3NWR cited_by UOGODPMX 6.0
RELATIONSHIP 3BURPWXO collaborated_with Y0AWLUUN 8.0
RELATIONSHIP ZHZYL5A9 affiliated_with 4QREPUTO 3.0
RELATIONSHIP J45F3NWR presented_at UERCIK82 1.5
RELATIONSHIP ZHZYL5A9 authored_by 2FU3NZR8 9.3
RELATIONSHIP ZHZYL5A9 authored_by J45F3NWR 0.5
RELATIONSHIP SXA506HI cited_by UERCIK82 4.4
RELATIONSHIP UERCIK82 collaborated_with UOGODPMX 1.1
RELATIONSHIP 3BURPWXO presented_at UERCIK82 3.0
RELATIONSHIP ZHZYL5A9 presented_at UERCIK82 2.7
RELATIONSHIP JQ753OIK presented_at 4QREPUTO 1.7
RELATIONSHIP SXA506HI presented_at Y0AWLUUN 1.4
RELATIONSHIP 4QREPUTO collaborated_with 2FU3NZR8 9.5
RELATIONSHIP 4QREPUTO cited_by SXA506HI 7.9
PRINT UOGODPMX
DELETE 2FU3NZR8
HIGHEST
DELETE J45F3NWR
DELETE UOGODPMX
PRINT Y0AWLUUN
DELETE Y0AWLUUN
PATH ZHZYL5A9 SXA506HI
PRINT SXA506HI
PRINT JQ753OIK
EXIT
