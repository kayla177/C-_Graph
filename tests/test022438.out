success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
J45F3NWR UERCIK82
success
JQ753OIK Y0AWLUUN 25
success
success
3BURPWXO SXA506HI
success
ZHZYL5A9 4QREPUTO SXA506HI 10.9
UERCIK82 4QREPUTO
4QREPUTO
